[FEAT]: Added common component to select number of rows per page .Added  functionality for liking and disliking of the  products along with the UI.

vinay352044 opened the pull request:
Added common component to select number of rows per page and integrated it to all the required components. Added 
functionality for liking and disliking of the 
products along with the UI.

jay-n-prajapati committed:
Merge branch 'UI-enhancement' of https://github.com/jay-n-prajapati/Bac-Mart into bug-navbar-profile

jay-n-prajapati committed:
Merge branch 'UI-enhancement' of https://github.com/jay-n-prajapati/Bac-Mart into bug-navbar-profile

meet-bhimani committed:
[REFACTOR(seller)]:added search and sort in seller products dashboard and added toast and error message for delete product from seller side

meet-bhimani committed:
[FIX(admin-products)]:added loader and toast messages for delete functionality in admin products dashboard

MansiJayswal committed:
[STYLE] : table component UI changes

vinay352044 committed:
[STYLE (pagination)] : Improved style for Pagination component

vinay352044 committed:
Merge branch 'UI-enhancement' of https://github.com/jay-n-prajapati/Bac-Mart into UI-Pagination-Searching-Filtering-Sorting-Whishlist

meet-bhimani committed:
Merge branch 'UI-enhancement' of https://github.com/jay-n-prajapati/Bac-Mart into bug-seller-dashboard

meet-bhimani committed:
[FIX(admin-products)]:added loader and toast messages for update functionality in admin products dashboard

jay-n-prajapati committed:
Merge pull request #31 from jay-n-prajapati/bug-seller-dashboard

[FIX]: added toast notifications and loader with error handling in seller products and admin products update and delete functionalities

vinay352044 committed:
Merge branch 'bug-fixing' of https://github.com/jay-n-prajapati/Bac-Mart into UI-Pagination-Searching-Filtering-Sorting-Whishlist

jay-n-prajapati committed:
Merge branch 'bug-fixing' into bug-table

jay-n-prajapati committed:
Merge pull request #33 from jay-n-prajapati/bug-table

[STYLE] : table component UI changes

jay-n-prajapati committed:
Merge branch 'bug-fixing' of https://github.com/jay-n-prajapati/Bac-Mart into bug-navbar-profile

jay-n-prajapati committed:
[FIX(conflict)] : conflict resolved in db.json file

jay-n-prajapati committed:
[REFACTOR] : minor changes in UI  profile page

jay-n-prajapati committed:
[STYLE] : input alignment changed

vinay352044 committed:
[FEAT]: Added common component to select number of rows per page and integrated it to all the required components. Added functionality for liking and disliking of the products along with the UI.

vinay352044 committed:
[FIXED]: Resolved merege conflicts in AdminCategories page, YourProducts page, AdminUsers page.

meet-bhimani reviewed (commented):
1. Remove console logs
2. Change the name of function to understandable format
3. Change value for 'rows per page' to appropriate options & change position for that option if it looks good somewhere else

meet-bhimani committed:
[FIX(footer)]: footer white space issue solved and position of footer is sticky at the bottom for every screen

vinay352044 committed:
[FIXED]: Removed unnecessary consoles from products component, renamed function as per naming convention in sorting component and eradicated the warning message coming while sorting..

jay-n-prajapati committed:
Merge pull request #35 from jay-n-prajapati/bug-footer

[FIX(footer)]: footer white space issue solved

jay-n-prajapati committed:
[FIX(conflict)] : conflict resolved from bug-footer branch in adminCategories

jay-n-prajapati committed:
[FIX] : wrong page request on profile page resolved

jay-n-prajapati committed:
[REFACTOR] : unneccessary console removed and typos corrected

jay-n-prajapati committed:
[MERGE #36] from bug-navbar-profile  --> bug-fixing

[REFACTOR] : wrong page request on profile resolved , UI changes in profile page , console removed

vinay352044 committed:
[FIXED]: Resolved merge conflicts in AdminCategories page.
